src/document.rs

```diff
@@ -47,6 +47,38 @@ fn yaml_to_liquid(yaml: &Yaml) -> Option<Value> {
     }
 }
 
+/// Escape HTML string
+///
+/// The function has been extracted from `librustdoc/html/escape.rs`.
+/// See also http://stackoverflow.com/questions/7381974 for details.
+pub fn escape_html(html: &str) -> String {
+    let mut escaped = String::new();
+    let mut last = 0;
+    for (i, ch) in html.bytes().enumerate() {
+        match ch as char {
+            '<' | '>' | '&' | '\'' | '"' => {
+                escaped.push_str(&html[last..i]);
+                let s = match ch as char {
+                    '>' => "&gt;",
+                    '<' => "&lt;",
+                    '&' => "&amp;",
+                    '\'' => "&#39;",
+                    '"' => "&quot;",
+                    _ => unreachable!(),
+                };
+                escaped.push_str(s);
+                last = i + 1;
+            }
+            _ => {}
+        }
+    }
+
+    if last < html.len() {
+        escaped.push_str(&html[last..]);
+    }
+    escaped
+}
```

Comment:
@johannhof 

* publishing a single function crate doesn't make much sense to me
* marksman_escape doesn't seem to be actively developed/supported
* adding marksman_escape as a dep for one simple function looks like an overkill

if you need it in liquid-rust, does it make sense to add this function there and reuse it in cobalt?